src/app/(dashboard)/residents/[id]/page.tsx

```
@@ -954,22 +826,36 @@
             {showDeleteConfirm && (
               <div className="fixed inset-0 z-50 overflow-y-auto">
                 <div className="flex min-h-full items-end justify-center p-4 text-center sm:items-center sm:p-0">
-                  <div className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" onClick={() => setShowDeleteConfirm(false)} />
+                  <div
+                    className="bg-opacity-75 fixed inset-0 bg-gray-500 transition-opacity"
+                    onClick={() => setShowDeleteConfirm(false)}
+                  />
                   <div className="relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-lg dark:bg-gray-800">
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)